src/pricing/index.njk

```diff
@@ -160,7 +160,7 @@ hubspot:
                             </div>
                             <a class="md:self-end ff-btn ff-btn--primary uppercase align-baseline w-full"
                                 href="{% if cloud %}{{ site.appURL }}/account/create{% else %}/contact-us{% endif %}"
-                                onclick="{% if cloud %}capture('cta-join', {'position': 'primary'}, {'plan': 'cloud-team'}){% else %}capture('cta-contact-us', {'position': 'primary'}, {'plan': 'sh-team'}){% endif %}">
+                                onclick="{% if cloud %}capture('cta-join', {'position': 'primary'}, {'plan': 'cloud-pro'}){% else %}capture('cta-contact-us', {'position': 'primary'}, {'plan': 'sh-pro'}){% endif %}">
```

Comment:
```suggestion
                                onclick="{% if cloud %}capture('cta-join', {'position': 'primary'}, {'plan': 'cloud-team'}){% else %}capture('cta-contact-us', {'position': 'primary'}, {'plan': 'sh-team'}){% endif %}">
```

Don't want to change these otherwise we lose historical data